docker-compose.yml

```diff
@@ -87,3 +87,16 @@ services:
     ports:
       - 8888:8080
 
+  api:
+    depends_on:
+      - db
+    image: hnskde/mong-api:develop
+    restart: "no"
+    environment:
+      DB_HOST: db
+      DB_NAME: imongr
+      DB_PWD: imongr
+      DB_USR: imongr
+      ORIGIN: blablabla
```

Comment:
Skal det stå noe annet her?

Reply:
Nei. Det er en sjekk i den delen av api-et som handler om cms-pålogging, som må ha env ORIGIN som ikke er tom. Burde fikses...

Reply:
Dette øverst i `apps/api/src/CMS.ts`:
```ts
if ("".match(originPattern)) {
  console.warn(
    "Insecure ORIGIN pattern used. This can give unauthorized users access to your repository.",
  );
  if (process.env.NODE_ENV === "production") {
    console.error("Will not run without a safe ORIGIN pattern in production.");
    process.exit();
  }
}
```